Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -129,6 +129,12 @@ to create a Kubernetes cluster locally or using a managed Kubernetes service:
           curl -LO \ |SCM_WEB|\/Documentation/installation/kind-config.yaml
           kind create cluster --config=kind-config.yaml
 
+.. note::
+
+   Cilium may fail to deploy due to too many open files in one or more
+   of the agent pods. If you notice this error, you can increase the
+   ``inotify`` resource limits on your host machine (see
+   `Pod errors due to "too many open files" <https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files>`__).
```

Comment:
Looks like during the change the newline got cutoff here, can you add it back in?

```suggestion
   `Pod errors due to "too many open files" <https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files>`__).

```